subsys/bluetooth/services/Kconfig.bas

```diff
@@ -0,0 +1,50 @@
+# Kconfig - Bluetooth GATT Battery service
+#
+# Copyright (c) 2018 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+menuconfig BT_GATT_BAS
+	bool "Enable BLE GATT Battery service"
+	select SENSOR
+	default n
+
+if (BT_GATT_BAS)
+
+choice
+	prompt "Battery services logging level"
+	default BT_GATT_BAS_LOG_LEVEL_INF
+
+ config BT_GATT_BAS_LOG_LEVEL_OFF
+	bool "Off"
+config BT_GATT_BAS_LOG_LEVEL_ERR
+	bool "Error"
+config BT_GATT_BAS_LOG_LEVEL_WRN
+	bool "Warning"
+config BT_GATT_BAS_LOG_LEVEL_INF
+	bool "Info"
+config BT_GATT_BAS_LOG_LEVEL_DBG
+	bool "Debug"
+
+ endchoice
+
+ config BT_GATT_BAS_LOG_LEVEL
+	int
+	default 0 if BT_GATT_BAS_LOG_LEVEL_OFF
+	default 1 if BT_GATT_BAS_LOG_LEVEL_ERR
+	default 2 if BT_GATT_BAS_LOG_LEVEL_WRN
+	default 3 if BT_GATT_BAS_LOG_LEVEL_INF
+	default 4 if BT_GATT_BAS_LOG_LEVEL_DBG
```

Comment:
Same comment as in the DIS: This is inconsistent with how it is done in all other drivers and subsystems

Reply:
yeah, I definitely see the advantages, I just don't like that the same functionality is differently defined across the whole tree...